config/toml.go

```diff
@@ -449,6 +449,8 @@ chunk-request-timeout = "{{ .StateSync.ChunkRequestTimeout }}"
 # The number of concurrent chunk and block fetchers to run (default: 4).
 fetchers = "{{ .StateSync.Fetchers }}"
 
+verify-light-block-timeout = "{{ .DBSync.VerifyLightBlockTimeout }}"
```

Comment:
Why is state sync use db sync config? Shall we add a state sync default config as well?

Reply:
oops copy paste error. Fixed